tests/integrationtest/l4_crd_test.go

```diff
@@ -95,13 +95,15 @@ func TestCreateDeleteL4RuleInvalidLBClass(t *testing.T) {
 	// this test checks the following scenario:
 	// adding a valid l4rule crd annotation to an invalid LBSvc should not cause the VS to come up
 	g := gomega.NewGomegaWithT(t)
-	L4RuleName := "test-l4rule"
+	L4RuleName := objNameMap.GenerateName("test-l4rule")
+	svcName := objNameMap.GenerateName(SINGLEPORTSVC)
+	modelName := MODEL_REDNS_PREFIX + svcName
```

Comment:
where is MODEl_REDNS_PREFIX defined ?
I couldn't find this in lib.go

Reply:
defined in l4_service_test.go